src/utils/BrowserUtilities.ts

```diff
@@ -19,15 +19,26 @@ import { getUserAgentRegExp } from "browserslist-useragent-regexp";
  */
 
 import { ReaderConfig } from "../navigator/IFrameNavigator";
+import * as HTMLUtilities from "./HTMLUtilities";
 
 /** Returns the current width of the document. */
 export function getWidth(): number {
-  return document.documentElement.clientWidth;
+  const wrapper = HTMLUtilities.findRequiredElement(
+    document,
+    "#iframe-wrapper"
+  ) as HTMLDivElement;
+
+  return wrapper.clientWidth;
 }
 
 /** Returns the current height of the document. */
 export function getHeight(): number {
-  return document.documentElement.clientHeight;
+  const wrapper = HTMLUtilities.findRequiredElement(
+    document,
+    "#iframe-wrapper"
+  ) as HTMLDivElement;
```

Comment:
Same

Reply:
:) same as the other response, this will be refactored.